### PR TITLE
Fix screwed up tests

### PR DESCRIPTION
**Pull request by WojciechBarczynski**

### CODE REVIEW

**Review by Janix4000 — CHANGES_REQUESTED** [review]
You need to update videos' positions after this change (no further need to divide caps dimensions): https://github.com/membraneframework-labs/membrane_video_compositor_plugin/blob/5c2619e7f0ada2d40b186e221688e90bc5408658/test/pipeline_integration_test.exs#L65-L70

Additionally, it is an excellent opportunity to change the names of output videos to correspond real resolutions of the composed videos.

File: test/composing_test.exs [hunk 1]
```diff
@@ -19,16 +19,16 @@ defmodule Membrane.VideoCompositor.Test.Composing do
     aligned: true
   }
 
-  describe "Checks composition and raw video pipeline on" do
+  describe "Checks composition and raw video pipeline on merging four" do
     @describetag :tmp_dir
```
Comment (Janix4000) [inline test/composing_test.exs:23]:
```suggestion
  describe "Checks composition and raw video pipeline on merging four videos on 2x2 grid" do
    @describetag :tmp_dir
```

File: test/composing_test.exs [hunk 2]
```diff
@@ -19,16 +19,16 @@ defmodule Membrane.VideoCompositor.Test.Composing do
...
 
     @tag wgpu: true
-    test "3s 720p 1fps raw video", %{tmp_dir: tmp_dir} do
+    test "3s 720p 1fps raw videos", %{tmp_dir: tmp_dir} do
```
Comment (Janix4000) [inline test/composing_test.exs:26]:
```suggestion
    test "3s 720p 1fps raw", %{tmp_dir: tmp_dir} do
```

File: test/composing_test.exs [hunk 3]
```diff
@@ -19,16 +19,16 @@ defmodule Membrane.VideoCompositor.Test.Composing do
...
       test_raw_composing(@hd_video, 3, tmp_dir, "short_videos")
     end
 
     @tag long_wgpu: true
-    test "10s 720p 1fps raw video", %{tmp_dir: tmp_dir} do
+    test "10s 720p 1fps raw videos", %{tmp_dir: tmp_dir} do
```
Comment (Janix4000) [inline test/composing_test.exs:31]:
```suggestion
    test "10s 720p 1fps raw", %{tmp_dir: tmp_dir} do
```

File: test/pipeline_integration_test.exs [hunk 4]
```diff
@@ -26,34 +26,34 @@ defmodule Membrane.VideoCompositor.Test.Pipeline do
     aligned: true
   }
 
-  describe "Checks h264 pipeline on" do
+  describe "Checks h264 pipeline on merging four" do
```
Comment (Janix4000) [inline test/pipeline_integration_test.exs:29]:
```suggestion
  describe "Checks h264 pipeline on merging four videos on 2x2 grid" do
```

File: test/pipeline_integration_test.exs [hunk 5]
```diff
@@ -26,34 +26,34 @@ defmodule Membrane.VideoCompositor.Test.Pipeline do
...
     @describetag :tmp_dir
 
     @tag wgpu: true
-    test "2s 720p 30fps video", %{tmp_dir: tmp_dir} do
+    test "2s 720p 30fps videos", %{tmp_dir: tmp_dir} do
```
Comment (Janix4000) [inline test/pipeline_integration_test.exs:33]:
```suggestion
    test "2s 720p 30fps h264", %{tmp_dir: tmp_dir} do
```

File: test/pipeline_integration_test.exs [hunk 6]
```diff
@@ -26,34 +26,34 @@ defmodule Membrane.VideoCompositor.Test.Pipeline do
...
       test_h264_pipeline(@hd_video, 2, "short_videos", tmp_dir)
     end
 
     @tag wgpu: true
-    test "1s 1080p 30fps video", %{tmp_dir: tmp_dir} do
+    test "1s 1080p 30fps videos", %{tmp_dir: tmp_dir} do
```
Comment (Janix4000) [inline test/pipeline_integration_test.exs:38]:
```suggestion
    test "1s 1080p 30fps h264", %{tmp_dir: tmp_dir} do
```

File: test/pipeline_integration_test.exs [hunk 7]
```diff
@@ -26,34 +26,34 @@ defmodule Membrane.VideoCompositor.Test.Pipeline do
...
       test_h264_pipeline(@full_hd_video, 1, "short_videos", tmp_dir)
     end
 
     @tag long_wgpu: true, timeout: 1_000_000
-    test "30s 720p 30fps video", %{tmp_dir: tmp_dir} do
+    test "30s 720p 30fps videos", %{tmp_dir: tmp_dir} do
```
Comment (Janix4000) [inline test/pipeline_integration_test.exs:43]:
```suggestion
    test "30s 720p 30fps h264", %{tmp_dir: tmp_dir} do
```

File: test/pipeline_integration_test.exs [hunk 8]
```diff
@@ -26,34 +26,34 @@ defmodule Membrane.VideoCompositor.Test.Pipeline do
...
       test_h264_pipeline(@hd_video, 30, "long_videos", tmp_dir)
     end
 
     @tag long_wgpu: true, timeout: 1_000_000
-    test "60s 1080p 30fps video", %{tmp_dir: tmp_dir} do
+    test "60s 1080p 30fps videos", %{tmp_dir: tmp_dir} do
```
Comment (Janix4000) [inline test/pipeline_integration_test.exs:48]:
```suggestion
    test "60s 1080p 30fps h264", %{tmp_dir: tmp_dir} do
```

**Review by Janix4000 — CHANGES_REQUESTED** [review]
I renamed tests a little.

You can batch all of the suggestions into one commit.

File: test/composing_test.exs (hunk 1 above)
Comment (WojciechBarczynski) [reply test/composing_test.exs]: done

File: test/composing_test.exs (hunk 2 above)
Comment (WojciechBarczynski) [reply test/composing_test.exs]: done

File: test/composing_test.exs (hunk 3 above)
Comment (WojciechBarczynski) [reply test/composing_test.exs]: done

File: test/pipeline_integration_test.exs (hunk 4 above)
Comment (WojciechBarczynski) [reply test/pipeline_integration_test.exs]: done

File: test/pipeline_integration_test.exs (hunk 7 above)
Comment (WojciechBarczynski) [reply test/pipeline_integration_test.exs]: done

**Review by Janix4000 — APPROVED** [review]
Gooood...

**Pull request merged**